charts/newrelic-logging/values.yaml

```diff
@@ -37,6 +37,68 @@ fluentBit:
   # - name: HTTPS_PROXY
   #   value: http://example.com:3128
 
+  # New Relic default configuration for fluent-bit.conf (service, inputs, filters, outputs)
+  # and parsers.conf (parsers). The configuration below is not configured for lowDataMode and will
+  # send all attributes.  If custom configuration is required, update these variables.
+  config:
+    service: |
+      [SERVICE]
+          Flush         1
+          Log_Level     ${LOG_LEVEL}
+          Daemon        off
+          Parsers_File  parsers.conf
+          HTTP_Server   On
+          HTTP_Listen   0.0.0.0
+          HTTP_Port     2020
+
+    inputs: |
+      [INPUT]
+          Name              tail
+          Tag               kube.*
+          Path              ${PATH}
+          Parser            ${LOG_PARSER}
+          DB                ${FB_DB}
+          Mem_Buf_Limit     7MB
+          Skip_Long_Lines   On
+          Refresh_Interval  10
+
+    filters: |
```

Comment:
I would suggest to have a `lowDataModeFilters` section here, that includes the `FILTER` configuration for `lowDataMode`, and just refer to it in the `configmap.yml` file.